deploy/olm-catalog/kubevirt-hyperconverged/1.4.0/hco00.crd.yaml

```diff
@@ -151,9 +151,11 @@ spec:
                 description: BareMetalPlatform indicates whether the infrastructure is baremetal.
                 type: boolean
               featureGates:
-                additionalProperties:
-                  type: boolean
-                description: featureGates HyperConvergedFeatureGates contain a list of feature enabler flags. Setting a flag to `true` will enable the feature. Setting `false` or removing the feature gate, disables the feature. optional+
+                description: featureGates HyperConvergedFeatureGates contains a list of feature enabler flags. Setting a flag to `true` will enable the feature. Setting `false` or removing the feature gate, disables the feature.
```

Comment:
What is `HyperConvergedFeatureGates` ? Is it something that a typical admin understands (I don't).
I don't believe it is a list, do I think that s better description would be "featureGates is a map of feature enabler flags.

Is it no longer optional? Do I have to add it to all RC?